modin/numpy/arr.py

```diff
@@ -2179,9 +2179,9 @@ def argmax(self, axis=None, out=None, keepdims=None):
                 raise numpy.AxisError(1, 1)
             if self._query_compiler.isna().any(axis=1).any(axis=0).to_numpy()[0, 0]:
                 na_row_map = self._query_compiler.isna().any(axis=1)
-                result = na_row_map.idxmax().to_numpy()[0, 0]
+                result = na_row_map.idxmax()
             else:
-                result = self._query_compiler.idxmax(axis=1)
+                result = self._query_compiler.idxmax(axis=0)
```

Comment:
Why was the axis changed?

Reply:
The opposite axis was incorrect. Here, query_compiler is an Nx1 array, and we need axis=0 to reduce it to 1x1. This was apparently already fixed for argmin on [line 2255](https://github.com/modin-project/modin/pull/5820/files#diff-a5565cfb004b107c88dbae43a112a1821d007fdb560eef82c66c25d57c73cb4fR2255).

Reply:
I see, thanks!